### PR TITLE
Adding controller to add petty replies to incorrect "I feel bad" messages

### PR DESCRIPTION
**Pull request by ohmycthulhu**
This PR adds new controller (similar to greetings) that listens to `I feel myself` or `I'm feeling myself` message and replies in a thread some cheesy lines. I ran out of ideas on replies, so new lines are welcomed. 

[Idea source](https://datarockets.slack.com/archives/C09V59MPS/p1677166314753969)

### CODE REVIEW

**Comment by npupko** [comment]
Damn. You really did it 😸

File: features/feelingMyself/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+} = require('../../lib')
+
+const shouldMessage = (message) => {
+  const notFromThread = !messageFromAnyThread(message)
+  const fromAllowableChannel = messageFromGeneralChannel(message)
+
+  return fromAllowableChannel && notFromThread
+}
+
+const REGEXP = /\bfeel(ing)? myself\b/ig
```
Comment (nikitasakau) [inline features/feelingMyself/index.js]: We could potentially also add "feels(ing) herself/himself" here, but in that case we would have to complicate the logic for replying ("I bet he is!"), so I think that's fine to go with `myself` for MVP 🙂

**Review by nikitasakau — APPROVED** [review]
Looks good 🙂

File: features/feelingMyself/index.js (hunk 1 above)
Comment (ohmycthulhu) [reply features/feelingMyself/index.js]: Technically, we can use: `/feel(?:s|ing)? (my|her|him|our|your|them)sel(?:f|ves)/` and then extract and analyze the extracted group afterwards :thinking:. I think we can add it before merge (along with tests, of course)

**Comment by ohmycthulhu** [comment]
Added the feature with parsing other pronouns as well

**Pull request merged**